db/migrations/mysql/000089_add-channelid-to-reaction.up.sql

```diff
@@ -14,7 +14,7 @@ EXECUTE addColumnIfNotExists;
 DEALLOCATE PREPARE addColumnIfNotExists;
 
 
-UPDATE Reactions SET ChannelId = (select ChannelId from Posts where Posts.Id = Reactions.PostId);
+UPDATE Reactions SET ChannelId = (select ChannelId from Posts where Posts.Id = Reactions.PostId) WHERE ChannelId="";
```

Comment:
Good thinking.

Reply:
Indeed. Always keep in mind that queries need to be idempotent. Nice work :+1: